src/main/java/eod/Player.java

```diff
@@ -194,7 +194,6 @@ public void sendPlayerOrder(boolean isFirst) {
 
     public void summonObject(WarObject object, Point point) {
         getBoard().summonObject(object, point);
-        output.sendWarObjectSummoned(object, point);
```

Comment:
I don't know if there are problems in this line.

Reply:
Yes there are. Thanks.